docs/recipes/pug.md

```diff
@@ -34,27 +34,47 @@ $ npm install --save-dev gulp-pug
 Add this task to your `gulpfile.js`, it will compile `.pug` files to `.html` files in `.tmp`:
 
 ```js
-gulp.task('views', () => {
-  return gulp.src('app/*.pug')
+function pug() {
```

Comment:
The title says "`views`", is there any specific reason why you named it `pug` instead? I think it's better to keep the name generic because you might want to switch to some other preprocessor down the road.

```suggestion
function views() {
```